style.css

```diff
@@ -454,6 +441,59 @@ img:hover {
   .desktop {
     display: none;
   }
+
+  .nav-menu {
+    background-color: #3c3a39;
+    height: 800px;
+    width: 100%;
+    font-family: 'Inter', sans-serif;
+    position: absolute;
+
+    /* opacity: 0.9; */
+    margin: 0;
+    padding: 0;
+    left: -200%;
+  }
+
```

Comment:
- [ ] [OPTIONAL] Please make sure your design isn't overflowing 🙏 

![20220718-165451_capture](https://user-images.githubusercontent.com/74814780/179552953-0528e48d-24c3-4898-bc0d-d90efd7f105f.gif)

Reply:
@Ugboaja-Uchechi i would gladly work on it, but i have tried everything i know possible, but couldn't fix it, could you guide me through it